test/api/find-or-create.test.js

```diff
@@ -43,27 +43,3 @@ test('findOrCreate', (t) => {
     })
   })
 })
-
-test('findOrCreate (metadata.recps)', (t) => {
```

Comment:
just to be sure I'm reading this right, we don't have any tests for encrypted metafeeds after this commit? AH yes, this is now https://github.com/ssbc/ssb-meta-feeds/pull/88. Okay makes sense 👍